packages/shared/src/components/plus/PlusFreeTrialAlert.tsx

```diff
@@ -0,0 +1,13 @@
+import type { ReactElement } from 'react';
+import React from 'react';
+import { Typography, TypographyType } from '../typography/Typography';
+
+export const PlusFreeTrialAlert = (): ReactElement => {
+  return (
+    <div className="min-w-full bg-status-success px-6 py-4 text-center text-black">
```

Comment:
You sure you want hardcoded text-black here?
Keep in mind it's doesn't respect light/dark mode.

Reply:
Plus/Onboarding pages don't seem to support light/dark mode but I've changes to be future proof ✔️